io.github.everestapi.Olympus.yml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.everestapi.Olympus
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: olympus
+finish-args:
+  # Allow the GUI to render
+  - --socket=x11
+  - --share=ipc
+  # Needed to access mod list and download mods
+  - --share=network
+  # Allow OpenGL acceleration
+  - --device=dri
+  # Needed to get the sound working
+  - --socket=pulseaudio
+  # Needed to launch celeste from Olympus
+  - --talk-name=org.freedesktop.Flatpak
+  # Keep the usual functions of Olympus working
+  - --filesystem=home
+  - --filesystem=~/.var/app/com.valvesoftware.Steam/data/Steam/steamapps/common/Celeste:rw
+  # Steam Deck micro sd path
+  - --filesystem=/run/media/mmcblk0p1/steamapps/common/Celeste:rw
+
+cleanup-commands:
+  - rm /app/opt/olympus/love /app/opt/olympus/sharp/Olympus.Sharp.bin.osx /app/opt/olympus/olympus.png /app/opt/olympus/olympus.desktop /app/opt/olympus/install.sh /app/opt/olympus/io.github.everestapi.Olympus.metainfo.xml
+
+modules:
+  - shared-modules/luajit/luajit.json
+  
+  - name: libmodplug
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/modplug-xmms/libmodplug-0.8.9.0.tar.gz
+        sha256: 457ca5a6c179656d66c01505c0d95fafaead4329b9dbaa0f997d00a3508ad9de
+
+  - name: love2d
+    sources:
+      - type: archive
+        url: https://github.com/love2d/love/releases/download/11.4/love-11.4-linux-src.tar.gz
+        sha256: e32feecaf5f4f210242a4b5f26cdf9ce9014bfde18d51286f643bb7fe51f5369 
+
+  - name: Olympus
+    buildsystem: simple
+    build-commands:
+      - install -d /app/bin /app/opt /app/share/application
+      - unzip dist.zip -d /app/opt/olympus
+      - ln -s /app/opt/olympus/olympus /app/bin/olympus
+      - install -Dm644 /app/opt/olympus/olympus.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 /app/opt/olympus/olympus.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key=Icon --set-value='io.github.everestapi.Olympus' /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 /app/opt/olympus/io.github.everestapi.Olympus.metainfo.xml /app/share/metainfo/io.github.everestapi.Olympus.metainfo.xml
```

Comment:
These two are required in appdata:

```
$ appstream-util validate io.github.everestapi.Olympus.metainfo.xml 
io.github.everestapi.Olympus.metainfo.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
```

Reply:
developer_name should also be there https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer_name

Reply:
For the <content_rating> I'm not sure how we can configure it since it's a mod loader and browser. You can theoretically load a mod that may contain something in the list

Reply:
Added here https://github.com/EverestAPI/Olympus/pull/77

Reply:
Should be fixed in the latest stable build